### PR TITLE
Add support for Bluetooth Xbox One controllers on macOS.

### PR DESCRIPTION
**Pull request by jechter**
https://fogbugz.unity3d.com/f/cases/1091502/

### CODE REVIEW

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - Support gamepad vibration on Switch.
+- Support for Bluetooth Xbox One controllers on macOS.
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: The usual conflict around CHANGELOG.md...

**Review by Rene-Damm — APPROVED** [review]

File: Packages/com.unity.inputsystem/CHANGELOG.md (hunk 1 above)
Comment (Rene-Damm) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: Needs to go up there in the 0.3-preview section.

Comment (jechter) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: Ah, yes. I just merged in github and did not even notice this.

**Review by Rene-Damm — APPROVED** [review]

**Review by douglas-piconi — APPROVED** [review]

**Comment by Rene-Damm** [comment]
@jechter Does this also work with 360 controllers?

**Comment by jechter** [comment]
@Rene-Damm This only affects wireless controllers on mac. AFAIK the only xbox controllers which can be wirelessly connected to macs are the latest generation of Xbox One controllers (which support Bluetooth, unlike the earlier models). Xbox (360 or other) controllers connected through a wire should still use the old code path.

**Comment by FRickReich** [comment]
as a sidenote, I would like to know how to actually use the xbox controller on Mac in the new system, because the default mapping is really strange (north button is horizontal axis?!)

**Comment by jechter** [comment]
@FRickReich can you tell me:
-which xbox controller (360 or xbox one), 
-how is it connected (usb or bluetooth)
-which version of the Input System package?

**Comment by FRickReich** [comment]
I am sorry, could have given these infos in the first run...
- Xbox One Controller from 2018
- bluetooth, but I tried both
- 0.2.6

**Comment by jechter** [comment]
Ok - so in that case, the problem you are describing is exactly the thing that this PR has fixed. Update to 0.2.8, and it should work as expected :)

**Comment by FRickReich** [comment]
sounds great! 0.2.8 is not in the package manager, so I guess I will have to manually download on GitHub?

Edit: ...excuse me. THE SECOND I wrote this, unity told me I can update the input system.

**Pull request merged**